models/topics.models.js

```diff
@@ -1,11 +1,27 @@
 const db = require("../db/connection");
 
 exports.selectTopics = () => {
-  return db.query("SELECT * FROM topics").then((topics) => {
+  return db.query("SELECT * FROM topics;").then((topics) => {
     return topics.rows;
   });
 };
 
+exports.selectArticles = () => {
+  return db
+    .query(
+      `SELECT article_id,
+  title,
+  topic,
+  author,
+  created_at,
+  votes,
+  article_img_url FROM articles ORDER BY created_at DESC;`
+    )
+    .then((articles) => {
```

Comment:
Will need a join,  group by and count to get out comment count here 👍

Reply:
Got it, thanks.